### DISCUSSION TITLE
Central drive

### DISCUSSION THREAD

**Discussion by SZEINER** [Leantime/leantime#1381] [Plugins]
Hello we work on plugin inspired by Google drive. 

<a href="https://ibb.co/ssjWKV1"><img src="https://i.ibb.co/ssjWKV1/central-drive.png" alt="central-drive" border="0"></a> <a href="https://ibb.co/Tb2GK35"><img src="https://i.ibb.co/Tb2GK35/Screenshot-2023-02-05-at-11-25-09-SZEINER-work-desk.png" alt="Screenshot-2023-02-05-at-11-25-09-SZEINER-work-desk" border="0"></a>

**Work:**

- Create folders under head project folder
- Upload files
- Permision from projects - you see only this folder where you have access
- Image preview and open
- Folder and file properties - you see date create and size for yet

**Future plan:**

- Better design
- Google drive integration
- Will be added...

**Comment by marcelfolaron** [comment]
Exciting, happy to work on the UI when you have something ready to share

**Comment by Reniezs** [comment]
- New interface of central drive.
- Added road folders

![CEF168BB-A649-493B-B532-84324E30DC57](https://user-images.githubusercontent.com/47902169/217368338-f953da3e-62fb-439f-8152-0b58b5d02d4f.jpeg)

**Comment by aboyington** [comment]
Is your plugin available for testing? I have been looking for something like this to organize my files

**Comment by SZEINER** [comment]
Hi Anthony, 

 

at the moment the plugins are not quite ready. We decided to spice it up a bit with interesting features, but at the moment we had to take some paid work at https://regiojet.com/ to cover the financial costs of the company. I hope you are well and the project is progressing.. We would like to help more but now we need to heal ourselves a little financially so that we can start properly again.

 

With regards,

 

-- 

Alexandr Szeiner CEO, Owner
SZEINER s.r.o
ICO: 05822882
www.szeiner.com <http://www.szeiner.com> 
Phone: +420 774 224 724
Mail: ***@***.*** ***@***.***>

**Comment by aboyington** [comment]
I understand, I also had to put things on hold to take on paid work. I look forward to your plugin when completed.
